### PR TITLE
build: Update to Next 13

### PR DESCRIPTION
**Pull request by ruioliveira02**

### CODE REVIEW

**Review by RuiL1904 — COMMENTED** [review]
image
Does this happen to you too?

File: layout/Attendee/Store/components/Product/index.tsx [hunk 1]
```diff
@@ -4,25 +4,25 @@ export default function Product({ name, id, image, price, enabled }) {
   return (
     <Link
       href={`/product/${id}`}
-      className={enabled ? "opacity-100" : "opacity-50"}
+      className={`group relative cursor-pointer ${
+        enabled ? "opacity-100" : "opacity-50"
+      }`}
       passHref
```
Comment (EnzoVieira) [inline layout/Attendee/Store/components/Product/index.tsx:10]: I think this passHref is no longer needed

https://nextjs.org/docs/pages/api-reference/components/link#if-the-child-is-a-custom-component-that-wraps-an-a-tag

File: layout/ResetPassword/components/ResetPasswordForm/index.tsx [hunk 2]
```diff
@@ -98,13 +98,11 @@ export default function ResetPasswordForm() {
           </p>
           <div className="w-96">
             <Link href="/login" passHref>
```
Comment (EnzoVieira) [inline layout/ResetPassword/components/ResetPasswordForm/index.tsx:100]: Here as well

File: components/Base/index.jsx [hunk 3]
```diff
@@ -68,17 +68,18 @@ const MobileNavbar = ({
...
+                  <Link
+                    key={item}
+                    href={`/manager/${item}`}
+                    className={classNames(
+                      item == href
+                        ? "bg-primary text-quinary"
+                        : "text-white hover:bg-primary hover:bg-opacity-50",
+                      "group flex items-center border-b-2 border-tertiary border-opacity-50 px-8 py-8 font-ibold text-xs"
+                    )}
+                    passHref
```
Comment (EnzoVieira) [inline components/Base/index.jsx:80]: Here

File: components/Base/index.jsx [hunk 4]
```diff
@@ -128,22 +129,27 @@ export default function Dashboard({
...
-                        : "text-white hover:bg-primary hover:bg-opacity-50",
-                      "group flex items-center border-b-2 border-tertiary border-opacity-50 px-8 py-8 font-ibold text-xs"
-                    )}
-                  >
-                    {item.toUpperCase()}
-                  </a>
+                <Link
+                  key={item}
+                  href={`${baseHref}/${item}`}
+                  passHref
```
Comment (EnzoVieira) [inline components/Base/index.jsx:144]: here

File: components/Dashboard/index.jsx [hunk 5]
```diff
@@ -142,8 +142,8 @@ export default function Dashboard(props) {
                 </a>
               ))}
             </nav>
-            <Link href="/" passHref>
-              <a className="px-4 font-iregular text-quinary">Log out 👋</a>
+            <Link href="/" className="px-4 font-iregular text-quinary" passHref>
```
Comment (EnzoVieira) [inline components/Dashboard/index.jsx:145]: here

File: components/Navbar/index.jsx [hunk 6]
```diff
@@ -58,25 +58,31 @@ export default function Navbar({ bgColor, fgColor, button, children }) {
...
                         {navigation.map((item) => (
-                          <Link key={item.slug} href={item.slug} passHref>
-                            <a className="font-iregular text-sm text-white text-opacity-40 hover:text-opacity-100">
-                              {item.name}
-                            </a>
+                          <Link
+                            key={item.slug}
+                            href={item.slug}
+                            className="font-iregular text-sm text-white text-opacity-40 hover:text-opacity-100"
+                            passHref
```
Comment (EnzoVieira) [inline components/Navbar/index.jsx:73]: here

File: components/Navbar/index.jsx [hunk 7]
```diff
@@ -58,25 +58,31 @@ export default function Navbar({ bgColor, fgColor, button, children }) {
...
                         {isAuthenticated ? null : (
-                          <Link key="login" href="/login" passHref>
-                            <a className="font-iregular text-sm text-white text-opacity-40 hover:text-opacity-100">
-                              Login
-                            </a>
+                          <Link
+                            key="login"
+                            href="/login"
+                            className="font-iregular text-sm text-white text-opacity-40 hover:text-opacity-100"
+                            passHref
```
Comment (EnzoVieira) [inline components/Navbar/index.jsx:83]: here

File: components/Navbar/index.jsx [hunk 8]
```diff
@@ -110,10 +120,12 @@ export default function Navbar({ bgColor, fgColor, button, children }) {
                               {user &&
                                 userNavigation(user.type).map((item) => (
                                   <Menu.Item key={item.name}>
-                                    <Link passHref href={item.slug}>
-                                      <a className="block px-4 py-2 text-sm text-gray-700 hover:bg-gray-100">
-                                        {item.name}
-                                      </a>
+                                    <Link
+                                      passHref
```
Comment (EnzoVieira) [inline components/Navbar/index.jsx:124]: here

File: components/Base/index.jsx (hunk 3 above)
Comment (EnzoVieira) [inline components/Base/index.jsx:80]: I think this `passHref` is no longer needed

https://nextjs.org/docs/pages/api-reference/components/link#if-the-child-is-a-custom-component-that-wraps-an-a-tag

File: components/Base/index.jsx (hunk 4 above)
Comment (EnzoVieira) [inline components/Base/index.jsx:144]: here as well

File: components/Dashboard/index.jsx (hunk 5 above)
Comment (EnzoVieira) [inline components/Dashboard/index.jsx:145]: here

File: components/Footer/index.tsx [hunk 9]
```diff
@@ -105,20 +104,33 @@ export default function Footer(props: IFooterProps) {
...
-              <a className="hover:underline">General Regulation</a>
+            <Link
+              passHref
+              href="/docs/regulamento.pdf"
+              className="hover:underline"
+            >
+              General Regulation
             </Link>
             <Link
               passHref
```
Comment (EnzoVieira) [inline components/Footer/index.tsx:122]: here

File: components/Footer/index.tsx [hunk 10]
```diff
@@ -105,20 +104,33 @@ export default function Footer(props: IFooterProps) {
...
               href="https://docs.google.com/forms/d/e/1FAIpQLSdV1bSyW2tcLuTC_jJCGdZ5NZHUlgETK7nQkOmyDzwb7eFS4Q/viewform"
+              className="hover:underline"
             >
-              <a className="hover:underline">Report a Problem</a>
+              Report a Problem
             </Link>
-            <Link passHref href="/docs/survival.pdf">
-              <a className="hover:underline">Survival Guide</a>
+            <Link
+              passHref
```
Comment (EnzoVieira) [inline components/Footer/index.tsx:129]: here

File: components/Navbar/index.jsx (hunk 7 above)
Comment (EnzoVieira) [inline components/Navbar/index.jsx:83]: here

File: components/Navbar/index.jsx (hunk 8 above)
Comment (EnzoVieira) [inline components/Navbar/index.jsx:124]: here

File: layout/Attendee/Store/components/Product/index.tsx (hunk 1 above)
Comment (EnzoVieira) [inline layout/Attendee/Store/components/Product/index.tsx:10]: here

File: layout/ResetPassword/components/ResetPasswordForm/index.tsx (hunk 2 above)
Comment (EnzoVieira) [inline layout/ResetPassword/components/ResetPasswordForm/index.tsx:100]: here

**Review by EnzoVieira — CHANGES_REQUESTED** [review]

**Review by EnzoVieira — DISMISSED** [review]
Great job! 🚀

**Review by MarioRodrigues10 — APPROVED** [review]
:+1:

**Pull request merged**